samples/EverythingServer/LoggingUpdateMessageSender.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using ModelContextProtocol;
+using ModelContextProtocol.Protocol.Types;
+using ModelContextProtocol.Server;
+
+namespace EverythingServer;
+
+public class LoggingUpdateMessageSender(IMcpServer server, Func<LoggingLevel> currentLevel) : BackgroundService
```

Comment:
Nit:

```suggestion
public class LoggingUpdateMessageSender(IMcpServer server, Func<LoggingLevel> getMinLevel) : BackgroundService
```

I'd argue the "currentLevel" is what's coming from `(LoggingLevel)Random.Shared.Next(_loggingLevelMap.Count)`, not this.